playbooks/ooi.yaml

```diff
@@ -0,0 +1,146 @@
+---
+- hosts: packstack
+  become: yes
+  become_method: sudo
+  tasks:
+    # XXX Hardcoded for OCCI 1.1
+    - name: Configure OOI repository
+      yum_repository:
+        name: ooi-centos-7-x86_6
+        description: Repository for OOI
+        baseurl: http://repository.egi.eu/community/software/ooi/occi-1.1/releases/centos/7/x86_64/RPMS/
+        gpgcheck: no
+        gpgkey: http://repository.egi.eu/community/keys/APPDBCOMM-DEB-PGP-KEY.asc
+
+    - name: Install OOI
+      yum: name=python-ooi state=installed
+
+    - name: Update nova api-paste.ini for OCCI
+      block:
+        - name: Add OCCI filter to nova api-paste.ini
+          ini_file:
+            path: /etc/nova/api-paste.ini
+            section: filter:occi
+            option: paste.filter_factory
+            value: ooi.wsgi:OCCIMiddleware.factory
+          notify:
+            - restart nova-api
+        - name: Add use to OCCI API composite to nova api-paste.ini
+          ini_file:
+            path: /etc/nova/api-paste.ini
+            section: composite:occi_api_11
+            option: use
+            value: call:nova.api.auth:pipeline_factory_v21
+          notify:
+            - restart nova-api
+        - name: Add noauth2 to OCCI API composite to nova api-paste.ini
+          ini_file:
+            path: /etc/nova/api-paste.ini
+            section: composite:occi_api_11
+            option: noauth2
+            # yamllint disable-line rule:line-length
+            value: cors http_proxy_to_wsgi compute_req_id faultwrap sizelimit osprofiler noauth2 occi osapi_compute_app_v21
+          notify:
+            - restart nova-api
+          tags:
+            - skip_ansible_lint
+        - name: Add keystone to OCCI API composite to nova api-paste.ini
+          ini_file:
+            path: /etc/nova/api-paste.ini
+            section: composite:occi_api_11
+            option: keystone
+            # yamllint disable-line rule:line-length
+            value: cors http_proxy_to_wsgi compute_req_id faultwrap sizelimit osprofiler authtoken keystonecontext legacy_v2_compatible occi osapi_compute_app_v21
```

Comment:
This is quite a long line ! I would keep this in a var file, so that you can manage it easier, and use `{{ keystone_vars }}` or something like that. Maybe keep this for when this becomes a role.

Reply:
Yes, I tried to also use the syntax with a > to allow to split the line, don't know exactly why but the task was changing/reloading all the time.
The fact is also that this line could change on different deployment, it's based on another value of the inifile. Need to find later a better way to manage this.

Reply:
can't you just do a `with_items: keystone_vars` ? then you would see which one specifically changed

Reply:
I think the problem was more a bug of ini_file with values split using > so not much we can address. As I said this shouldn't really be a static line, let's plan to update this in the roles and keep it like for the time being.